ckb-importer/src/main/java/com/hartwig/serve/ckb/json/drug/DrugDatamodelChecker.java

```diff
@@ -48,6 +48,8 @@ public static JsonDatamodelChecker referenceObjectChecker() {
         map.put("id", true);
         map.put("pubMedId", true);
         map.put("title", true);
+        map.put("shortJournalTitle",true);
+        map.put("pages",true);
```

Comment:
Formatting looks incorrect here (space required after comma). 

You should be able to auto-reformat